exonum/src/blockchain/config.rs

```diff
@@ -89,16 +89,19 @@ pub struct StoredConfiguration {
 /// [Consensus in Exonum](https://exonum.com/doc/architecture/consensus/).
 #[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
 pub struct ConsensusConfig {
-    /// Interval between rounds. This interval defines the time that passes
+    /// Interval between first two rounds. This interval defines the time that passes
     /// between the moment a new block is committed to the blockchain and the
-    /// time when a new round starts, regardless of whether a new block has
+    /// time when second round starts, regardless of whether a new block has
     /// been committed during this period or not.
+    /// Each consecutive round will be longer then previous by round_timeout_increase.
     ///
     /// Note that rounds in Exonum
     /// do not have a defined end time. Nodes in a new round can
     /// continue to vote for proposals and process messages related to previous
     /// rounds.
-    pub round_timeout: Milliseconds,
+    pub first_round_timeout: Milliseconds,
+    /// Amount of time that is added to round timeout as round number increase.
```

Comment:
I think `Amount of time that is added to the round timeout each round` is better.